sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -324,6 +324,16 @@ export class ProgramInfoRequestController extends BaseController {
         );
       }
 
+      await this.applicationService.validateOverlappingDatesAndPIR(
```

Comment:
Please add the `INVALID_OPERATION_IN_THE_CURRENT_STATUS` to the array of possible errors that will generate the `UnprocessableEntityException`.